specs/DevSpecs/TextRun.Underline.md

```diff
@@ -0,0 +1,95 @@
+# InputValidation
+
+## Proposal Link
+https://github.com/microsoft/AdaptiveCards/issues/3051
+
+## Dependencies
```

Comment:
I thought Andrew mentioned this in his proposal notes for this : 
<snip>  TextBlock aspect is dependent on the other convergence proposal #3071 /snip> ?